dist/min/darkmode.css

```diff
@@ -0,0 +1 @@
,.Checkbox:checked{background:linear-gradient(180deg,#212c3b,#090c10);border:1px solid #161d27;border-radius:3px}#activity-popup-dialog .modal-body,#auto-close p{background-color:#212c3b}.ProfileNameTruncated .UserBadges .Icon{padding-right:0!important;padding-left:2px}.MomentFormatLink-description,.MomentFormatLink-title{color:#657786}.MomentFormatLink--withText .MomentFormatLink-anchor:active,.MomentFormatLink--withText .MomentFormatLink-anchor:focus,.MomentFormatLink--withText .MomentFormatLink-anchor:hover,.MomentFormatLink-anchor:active,.MomentFormatLink-anchor:focus,.MomentFormatLink-anchor:hover{background-color:#161d27}button:focus{box-shadow:0 0 0 2px #1da1f2,0 0 2px 4px hsla(0,0%,100%,.25)!important}.TwitterCard .EdgeButton--secondary.disabled,.TwitterCard .EdgeButton--secondary[aria-disabled=true],.TwitterCard .EdgeButton--secondary[disabled],.TwitterCard .EdgeButton--secondary[disabled]:hover{background-color:transparent}.TwitterCard .EdgeButton--secondary{background:transparent}.TwitterCard .EdgeButton--secondary:hover{background-color:#1b95e0}.ResponsiveLayout{background:transparent!important}.ResponsiveLayout .PageContainer,.web-reporting-flow{background-color:transparent!important}.web-reporting-flow .page-bottom,.web-reporting-flow .reportFlowHeader-web,.web-reporting-flow .web-item{color:#fff}.reportFlowHeader-web .list-explanation.success-page{color:#fff!important}.AdaptiveNewsRelatedHeadlines{background-color:#212c3b;border:1px solid #2f3d51}}@-moz-document url-prefix("https://mobile.twitter.com"){._1_vKaW5Q,._1bfHukDN,._1d_6kzhv,._1L-Rr7sz,._1lW9BO8P,._1qygzEvp,._2EmxsDVG,._2jEwv4_q,._2nUWv41O,._2vwsIlm0,._3cKAE7hX,._3dCVClw0,._3lHGpXjt,._10x6nIYi,._16q9OV8F,._24u5-vsm,.FsR6j-G7,.FsR6j-G7:active,.FsR6j-G7:focus,.FsR6j-G7:hover,.Gpm7TAP3,.lIgYF7yz,.W1P7mM7E,html{background-color:#0e131a;color:#fff}._2vY0UT8z:active,._3bf6snf7,.qb_4uBPl{background-color:rgba(29,161,242,.15)}._1YVNMg8K ._29kyA7wX{background-color:#657786;border-color:#657786}._266FL8X1:before{border-color:#657786}._1y-7iMRs:active,._1y-7iMRs:focus,._1y-7iMRs:hover,.ktZMpANQ{background-color:rgba(101,119,134,.25)}._1n7-Ka2i{background:#0e131a;border-top:1px solid #657786}._1MWHskJm,._1NUqA_Ya,._38myriBx,._38myriBx:active,._38myriBx:focus,._38myriBx:hover{background-color:#1c9dec}._1u_kMV_N,._3WJqTbOE{color:#fff}._1d_6kzhv,._1s7qNPIc,._2jEwv4_q{box-shadow:0 1px 0 rgba(101,119,134,.25)}._1_vKaW5Q,._1gq1kIcW,._1Oxj60h6,._1qygzEvp,._1XHWlb5c,._2hx-WgQl,._2nUWv41O,._2OhV6WSs ._1eF_MiFx ._2sSKjr8W,._2SfRhtNW,._2Sszw7pC,._3-utd0lH,._3DRCrDG0,._3facEWb0,._3facEWb0:active,._3facEWb0:focus,._3facEWb0:hover,._16q9OV8F,._22uh2-tW,._33HzS5QJ,._qZ42ApI,.FaFY-n1u,.FsR6j-G7,.pQ0LYqFV,.qb_4uBPl{border-color:rgba(101,119,134,.25)}._1HXcreMa,._3ZSf8YGw{color:#1c9dec}._3fv5_7QL{border-bottom:1px solid #1b95e0}._1_KafmK5{border-bottom:1px solid rgba(101,119,134,.25)}._3Ko_vUO1,._15-p_S0N{border:1px solid rgba(101,119,134,.25);color:#999}._3QPrjhX2,.lIgYF7yz{border-color:transparent}}
```

Comment:
idk, I upload through the github website (too lazy to learn git) and drag the scss file and dist folder in and commit it.